guide/01-getting-started/using-the-jupyter-lab-environment.ipynb

```diff
@@ -36,21 +36,55 @@
     "\n",
```

Comment:
<p>For (3), nodejs has to be updated to newer versions, if already installed. (I had its version at 10.0.0, and the subsequent rebuild/build failed on me, until I have it updated to the latest 14.6.0)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/745/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>